src/components/dialogs/stashed-elements/stashed-elements-dialog.tsx

```diff
@@ -27,6 +27,7 @@ interface IStashedElementsDialog {
     onClose: () => void;
     stashedElements: any[];
     onStashedElementChange: () => any[];
+    activeDirectory?: IDirectory;
```

Comment:
The StashedElementsDialog component should not be responsible for determining the directory where the elements will be restored. It would be cleaner and more robust to pass the directory as a prop from the parent component, allowing the parent to specify which directory (selected or active) should be used.

Reply:
done